Plugin: 修仙_2.0

**插件名称：**

修仙_2.0

**插件功能：**

基于原版修仙的修改与重构

**PyPI 项目名：**

nonebot_plugin_xiuxian_2

**插件 import 包名：**

nonebot_plugin_xiuxian_2

**插件项目仓库/主页链接：**

https://github.com/QingMuCat/nonebot_plugin_xiuxian_2.0

**标签：**

[{"label":"文游","color":"#ea5252"},{"label":"修仙","color":"#4e9f9f"}]

# 📃 商店发布检查结果

> Plugin: xiuxian_2.0

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-xiuxian-2 的信息如下：
    name         : nonebot-plugin-xiuxian-2 
     version      : 2.1.5                    
     description  : 修仙插件                     
    
    dependencies
     - anyio 3.6.2
     - APScheduler 3.10.0
     - asttokens 2.2.1
     - certifi 2022.12.7
     - charset-normalizer *
     - cheap-repr 0.5.1
     - click 8.1.3
     - colorama 0.4.6
     - executing 1.2.0
     - fastapi 0.89.1
     - h11 0.14.0
     - httpcore 0.16.3
     - httptools 0.5.0
     - httpx 0.23.3
     - idna 3.4
     - loguru 0.6.0
     - msgpack 1.0.4
     - multidict 6.0.4
     - nonebot-adapter-onebot 2.2.1
     - nonebot-plugin-apscheduler 0.2.0
     - nonebot2 2.0.0rc3
     - numpy 1.23.5
     - Pillow 9.4.0
     - pydantic 1.10.4
     - Pygments 2.14.0
     - pygtrie 2.5.0
     - python-dotenv 0.21.1
     - pytz 2022.7.1
     - pytz-deprecation-shim 0.1.0.post0
     - PyYAML 6.0
     - rfc3986 1.5.0
     - sentry-sdk 1.14.0
     - six 1.16.0
     - sniffio 1.3.0
     - starlette 0.22.0
     - tomlkit 0.11.6
     - typing-extensions 4.4.0
     - tzdata 2022.7
     - tzlocal 4.2
     - ujson 5.7.0
     - urllib3 1.26.14
     - uvicorn 0.20.0
     - watchfiles 0.18.1
     - wcwidth 0.2.6
     - websockets 10.4
     - wget 3.2
     - win32-setctime 1.1.0
     - yarl 1.8.2
插件 nonebot-plugin-xiuxian-2 加载出错：
02-22 10:25:47 [SUCCESS] nonebot | NoneBot is initializing...
02-22 10:25:47 [INFO] nonebot | Current Env: prod
02-22 10:25:47 [ERROR] nonebot | Failed to import &quot;nonebot-plugin-xiuxian-2&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot-plugin-xiuxian-2&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named &#x27;nonebot-plugin-xiuxian-2&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: 文游-#ea5252, 修仙-#4e9f9f。</li><li>✅ 项目 <a href="https://github.com/QingMuCat/nonebot_plugin_xiuxian_2.0">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-xiuxian-2/">nonebot-plugin-xiuxian-2</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4417,5 +4417,24 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_xiuxian_2",
+    "project_link": "nonebot_plugin_xiuxian_2",
+    "name": "修仙_2.0",
+    "desc": "基于原版修仙的修改与重构",
+    "author": "QingMuCat",
+    "homepage": "https://github.com/QingMuCat/nonebot_plugin_xiuxian_2.0",
+    "tags": [
+      {
+        "label": "文游",
+        "color": "#ea5252"
+      },
+      {
+        "label": "修仙",
+        "color": "#4e9f9f"
+      }
+    ],
+    "is_official": false
   }
 ]

